backend/src/schema/resolvers/registration.spec.js

```diff
@@ -351,7 +352,7 @@ describe('SignupVerification', () => {
         name: 'John Doe',
         password: '123',
         email: 'john@example.org',
-        termsAndConditionsAgreedVersion: '0.0.1',
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-12T15:31:23Z" title="Thursday, September 12th 2019, 5:31:23 pm +02:00">Sep 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -444,11 +446,11 @@ describe('SignupVerification', () => {
             expect(emails).toHaveLength(1)
           })
 
-          it('is version of terms and conditions saved correctly', async () => {
+          it('if a current date of the General Terms and Conditions is available', async () => {
```


```suggestion
          it('updates `termsAndConditionsAgreedAt`'', async () => {
```

The `it` is the **test case** and `describe` most of the times describes the test **setup**. Therefore it's good practice to explain **what it does** in the test case description.

Or why is the word *if* in the description? Are you unsure what the test is doing @ogerly ? :stuck_out_tongue_closed_eyes: